### PR TITLE
[swss]: Fix missing comma in the ports.json.j2 file

### PR DESCRIPTION
**Pull request by stcheng**
Add J2 test case as well as a sample output

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

**- How to verify it**
Run the pytest

**- Description for the changelog**
this issue was introduced here: https://github.com/Azure/sonic-buildimage/pull/1207

### CODE REVIEW

File: dockers/docker-orchagent/ports.json.j2 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {% if PORT[port].has_key('speed') %}
     {
         "PORT_TABLE:{{ port }}": {
-            "speed": "{{ PORT[port]['speed'] }}"
+            "speed": "{{ PORT[port]['speed'] }}",
```
Comment (rodnymolina) [inline dockers/docker-orchagent/ports.json.j2]: Thanks for taking care of this one.

File: src/sonic-config-engine/tests/simple-sample-graph.xml [hunk 2]
```diff
@@ -217,6 +217,7 @@
           <PortName>0</PortName>
           <Priority>0</Priority>
           <Speed>10000</Speed>
+          <Description>Interface description</Description>
```
Comment (rodnymolina) [inline src/sonic-config-engine/tests/simple-sample-graph.xml]: I thought about mixing interfaces with and without "description" to make sure that our code/testcases can deal with both scenarios.

Comment (stcheng) [reply src/sonic-config-engine/tests/simple-sample-graph.xml]: Sure. I'll remove two of them.

Comment (stcheng) [reply src/sonic-config-engine/tests/simple-sample-graph.xml]: Done

**Pull request merged**